game/src/js/objects/Character.js

```diff
@@ -200,4 +200,51 @@ export default class Character extends Phaser.Physics.Arcade.Sprite {
       character.destroy();
     }
   }
-}
\ No newline at end of file
+  
+  // sets the character to use an AI, stalking and aiming at the player.
+  // doesn't deal with multiple opponents yet.
+  setAI(player) {
+    this.AIOn = true;
+    this.targetPlayer = player;
+
+  }
+
+  // turns off the current AI.
+  turnAIOff() {
+    this.AIOn = false;
+    this.targetPlayer = null;
+  }
+
+  // performs one tick worth of time of what the AI is going to do.
+  updateAI() {
+    if (!this.AIOn) {
+      return;
+    }
+    const targetXPosition = this.targetPlayer.x;
+    const targetYPosition = this.targetPlayer.y;
+    const xDifference = this.x - targetXPosition;
+    const yDifference = this.y - targetYPosition;
+    int distance = Math.sqrt(xDifference * xDifference + yDifference * yDifference);
```

Comment:
Looks like this code was adapted from another language but is not yet fully converted. This `int` should probably be a `let` or a `const`. However, it doesn't look like this variable is used.

Reply:
Yeah, sorry way too used to using Java.  Fixed.